.deliveroo-local/README.md

```diff
@@ -0,0 +1,31 @@
+# routemaster
+
+## Development
+
+The project supports the following development targets:
+
+```
+make build  // Builds the docker-compose environment.
+make start  // Starts the application.
+make test   // Runs tests.
+```
+
+## Configuration
+
+Use [the `rtm`
+client](https://github.com/deliveroo/routemaster-client#cli-usage) to configure
+this instance.
+
+Add the following to `~/.rtmrc`:
+
+```
+local:
+  bus:   routemaster.deliveroo-local.com
+  token: routemaster
+```
+
+Run `rtm` with the `@local` environment:
+
+```
+rtm -b @local <command>
```

Comment:
This is my bad; this _correct_ syntax is `rtm <command> -b @local`. The CLI client expects flag options after the command.